pm/sendermonitor.go

```diff
@@ -327,6 +327,8 @@ func (sm *LocalSenderMonitor) startCleanupLoop() {
 	for {
 		select {
 		case <-ticker.C:
+			sm.smgr.UpdateSenderInfos()
```

Comment:
Refresh at new round start makes sense to me.  Looks like it is already looking for updated information at the start of each round. Did not see that my first time through it!

https://github.com/livepeer/go-livepeer/blob/86b248dc6cb445132aa84ecf7440ecf71137dafe/eth/watchers/senderwatcher.go#L280

Reply:
Since `handleRoundEvent` already calls `GetSenderInfo` and `ClaimedReserve` on a new round, should these additions simply be dropped from `startCleanupLoop`?